thorlcr/graph/thgraph.hpp

```diff
@@ -121,14 +121,23 @@ interface IThorResult : extends IInterface
     virtual const void * getLinkedRowResult() = 0;
 };
 
+enum ThorGraphResultType:unsigned
+{
+    thorgraphresult_nul = 0x00,
+    thorgraphresult_distributed = 0x01,
+    thorgraphresult_allownulls=0x02,
```

Comment:
This looks likely to cause problem (and I think there is one above).  Better would be

    thorgraphresult_grouped = 0x02,
    thorgraphresult_sparse = 0x04
    thorgraphresult_allownulls= (thorgraphresult_grouped|thorgraphresult_sparse),